### DISCUSSION TITLE
Policies included in Azure landing zones reference implementations

### DISCUSSION THREAD

**Discussion by sweprs** [Azure/Enterprise-Scale#1795] [General] [answered]
Regarding the impementation of: Policies included in Azure landing zones reference implementations
https://github.com/Azure/Enterprise-Scale/blob/main/docs/wiki/ALZ-Policies.md

We are trying to figure out the proper way to implement this using IAC and Bicep. We have built a custom module that takes for instance every policy in a policy set (initiative) and assigns it (providing the right parameters)
Looking to implement this at scale with the Landing Zone policies, we realise that most of the policy sets are custom and contains both built-in policies and ALZ policies. Meaning our approach is not viable but you have to break out the policies in custom iniatives.

Is the attention going forward that all iniatives should use built in policies or how does the future look?

Or is it a better way of doing this. Have read the documentation and cant find a "how to/best practice" using bicep when implementing ALZ landing zone policies

**Comment by jtracey93** [comment, marked as answer]
Hey, id highly suggest consuming our ALZ Bicep implementation where we maintain the default ALZ policies and assignments that you can take and customize as you need.

Checkout the repo: https://aka.ms/alz/bicep
